actors/builtin/miner/miner_state.go

```diff
@@ -131,6 +163,13 @@ func (st *State) GetSectorSize() abi.SectorSize {
 	return st.Info.SectorSize
 }
 
+// Returns the epoch that starts the current proving period, and whether that period starts on or after epoch 0.
+// The period start can be negative during the first proving period of the chain.
```

Comment:
for what case does this method return negative values? I don't see how that's possible given the impl of `quantizeDown`.